src/tests/useBoolean.test.ts

```diff
@@ -0,0 +1,54 @@
+import useBoolean from "@/hooks/useBoolean";
+import { renderHook, act, cleanup } from "@testing-library/react-hooks";
+import { afterEach, beforeEach, describe, expect, it } from "vitest";
+
+describe("useBoolean", () => {
+	beforeEach(() => {
+		cleanup();
+	});
```

Comment:
Why cleaning before AND after ? 

Just using after should be okay i think.

Let me know if i'm missing something